Marlin/src/pins/pins.h

```diff
@@ -522,7 +522,7 @@
 #elif MB(MKS_ROBIN_PRO)
   #include "stm32f1/pins_MKS_ROBIN_PRO.h"       // STM32F1                                env:mks_robin_pro env:mks_robin_pro_maple
 #elif MB(MKS_ROBIN_E3)
-  #include "stm32f1/pins_MKS_ROBIN_E3.h"        // STM32F1                                env:mks_robin_e3 env:mks_robin_e3_maple
+  #include "stm32f1/pins_MKS_ROBIN_E3.h"        // STM32F1                                env:mks_robin_e3
```

Comment:
Maple still exists for these boards: https://github.com/MarlinFirmware/Marlin/blob/5809bdd400c77ef1e25b5215ef14d07671b9aaf6/ini/stm32f1-maple.ini#L243-L253

Reply:
Legacy rubbish, it has to go!
I'm not updating for maple env when the non maple env is there and working and the only option for 3/4 boards

Reply:
Then the Maple environment should be removed for these boards.

Reply:
There are still some cases, apparently, where using Maple acts as a workaround for a bug in Marlin or maybe in the STM32 variant, or some combination thereof. There was some buzz about Maple getting a refresh, but don't quote me on that.